libraries/mpxe-store/src/store.c

```diff
@@ -45,6 +48,10 @@ static void prv_sigusr2(int signo) {
   pthread_mutex_unlock(&s_log_lock);
 }
 
+static void prv_sigrtmin(int signo) {
```

Comment:
lol isn't this gonna conflict with the gpio interrupts since they also use sigrt and use the full range of 1-32? maybe just bump this to SIGIO, what do you think?